test/conftest.py

```diff
@@ -1,16 +1,45 @@
 # flake8: noqa
+import boto3
 import pytest
 import os
+import requests
+
+from dcicutils.misc_utils import check_true
 from dcicutils.s3_utils import s3Utils
 from dcicutils.ff_utils import authorized_request
+from dcicutils.beanstalk_utils import describe_beanstalk_environments, REGION
+
+
+def _discover_es_info(envname):
```

Comment:
This utility is not needed - use `ff_utils.get_health_page`.

Reply:
I was trying to avoid using that because we're testing that. I did a similar thing with computing blue/green. My thought was that the real "strength" of a wired name, even though it has weaknesses, is that it specifies an anchor that other tests will judge consistency against. If it uses those other tools, it's at risk of them being wrong. And maybe that argues for failing on the throttling or using a more low-tech other throttling. It might make sense to call that other function and compare the result to make sure we agree we're talking to the right page. It seems like a hard thing for the get_health_page thing to get wrong, but I'm ignoring that and thinking more in terms of what are the consequences if it got the health page for the wrong server, and it seems like the consequence is that a lot of integration tests are done to the wrong resource, which could put that resource at some risk.

In general, I think testing code is not like regular code in that sometimes things are inelegant, redundant, differing in syntax and that can be good because otherwise if you're too regular you may not trip up some subtle difference. The core value in algebra as a concept, whether high school algebra, abstract algebra, or the kind of weird programming algebra we do is that no matter which way you go to get an answer, you want to get the same answer. But in programming that's an ideal more than a reality, so testing goodness by going different routes can sometimes be a virtue in itself because there is additional confidence in something tested several ways.

Reply:
I made it do a consistency check as part of the setup. I'm going to leave it at that. :)